pkgs/tools/admin/pgadmin/default.nix

```
@@ -0,0 +1,152 @@
+{ stdenv
+, lib
+, python3
+, fetchurl
+, zlib
+, python3Packages
+, mkYarnModules
+}:
+
+let
+
+  pname = "pgadmin";
+  version = "6.3";
+
+  src = fetchurl {
+    url = "https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v${version}/source/pgadmin4-${version}.tar.gz";
+    sha256 = "0xcg5gx6hf4k15idjkxzsnm5s9829fn4li54ga9qbyfqn3wwpg0h";
+  };
+
+  yarnDeps = mkYarnModules {
+    pname = "${pname}-yarn-deps";
+    inherit version;
+    packageJSON = ./package.json;
+    yarnLock = ./yarn.lock;
+    yarnNix = ./yarn.nix;
+  };
+in
+
+python3Packages.buildPythonApplication rec {
+  inherit pname version src;
+
+  # from Dockerfile
+  CPPFLAGS = "-DPNG_ARM_NEON_OPT=0";
+
+  format = "other";
+
+  postPatch = ''
+    # don't build sphinx documentation
+    sed -r 's|LC_ALL=en_US.UTF-8.* Makefile.sphinx .*$|true|g' -i Makefile
+    patchShebangs .
+    # relax dependencies
+    substituteInPlace requirements.txt \
+    --replace "Pillow==8.3.*" "Pillow>=8.3.0" \
+    --replace "psycopg2==2.8.*" "psycopg2>=2.8.0" \
+    --replace "cryptography==3.*" "cryptography>=3.0" \
+    --replace "requests==2.25.*" "requests>=2.25.0"
+    # don't use Server Mode (can be overridden later)
+    substituteInPlace pkg/pip/setup_pip.py \
+    --replace "req = req.replace('psycopg2', 'psycopg2-binary')" "req = req" \
+    --replace "builtins.SERVER_MODE = None" "builtins.SERVER_MODE = False"
+  '';
+
+  buildPhase = ''
+    # Adapted from pkg/pip/build.sh
+    echo Creating required directories...
+    mkdir pip-build
+
+    mkdir pip-build/pgadmin4
+    mkdir pip-build/pgadmin4/docs
+
+    # Build the clean tree
+    cd web
+    cp -r * ../pip-build/pgadmin4
+    cd ../docs
+    cp -r * ../pip-build/pgadmin4/docs
+    cd ../
+
+    cp -r ${yarnDeps}/* pip-build/pgadmin4
+
+    echo Creating distro config...
+    echo HELP_PATH = \'../../docs/en_US/_build/html/\' > pip-build/pgadmin4/config_distro.py
+    echo MINIFY_HTML = False >> pip-build/pgadmin4/config_distro.py
+
+    echo Creating manifest...
+    echo recursive-include pgadmin4 \* > pip-build/MANIFEST.in
+
+    echo Building wheel...
+    cd pip-build
+    ${python3}/bin/python ../pkg/pip/setup_pip.py bdist_wheel
+    cd ../
+
+    # Get the build
+    if [ ! -d dist ]; then
+        mkdir dist
+    fi
+    cp pip-build/dist/*.whl dist/
+  '';
+
+  installPhase = ''
+    ${python3}/bin/python -m pip install dist/*.whl --prefix $out
```

```suggestion
    ${python3.interpreter} -m pip install dist/*.whl --prefix $out
```
Also python3 should probably be added to nativeBuildInputs instead.